include/oneapi/dpl/pstl/onedpl_config.h

```diff
@@ -304,4 +304,11 @@
 #    define _ONEDPL_ICPX_OMP_SIMD_DESTROY_WINDOWS_BROKEN 0
 #endif
 
+// Avoid compile errors due incorrect implementation of std::bit_floor in MS STL: used "extern" variable __isa_available
+#if defined(_MSC_VER) && _ONEDPL_BACKEND_SYCL
```

Comment:
Its probably best to just avoid it for `_MSC_VER`, but if we want to tie it to a version, it looks like use of `__isa_available` began with this commit from 2020.  
https://github.com/microsoft/STL/pull/795/files#diff-3d3cc1113abed41109da08c2dc07230cd99c45de72dd05b85592e0c61308175b

I do not know if it worked in a kernel prior to that commit, or what version that corresponds to.

Reply:
My idea was as far as `__isa_available` is undocumented we shouldn't check `_MSC_VER` version number, good enough simple to disable it's usage inside Kernel code.

Reply:
Yeah, I think its not much of a "downgrade" to end up in the #else here anyway, so I'm OK with just `_MSC_VER`.